SecondYearFirstSemestr/MyNUnit/Tests/Tests.cs

```diff
@@ -0,0 +1,150 @@
+﻿using MyNUnit.Attributes;
+
+namespace MyNUnit
+{
+	public class Tests
+	{
+
+		[Attributes.Test(typeof(Exception), "123")]
+		public string IgnoreTest()
+		{
+			return "1";
+		}
+
+		[Attributes.Test(typeof(ArgumentNullException))]
+		public int WasntExpectedExceptionTest()
+		{
+			return 6;
+		}
+
+		[Attributes.Test(typeof(ArgumentNullException))]
+		public int CaughtExpectedExceptionTest()
+		{
+			throw new ArgumentNullException();
+		}
+
+		[Attributes.Test]
+		public void CaughtUnexpectedExceptionTest()
+		{
+			throw new Exception();
+		}
+	}
+
+	public class TestWithBeforeAttribute
+	{
+		[Before]
+		public static void Do()
+		{
+
+		}
+
+		[Attributes.Test]
+		public int Test()
+		{
+			return 6;
+		}
+	}
+	public class BadTestWithBeforeAttribute
+	{
+		[Before]
+		public static void Do()
+		{
+			throw new NotImplementedException();
+		}
+
+		[Attributes.Test]
+		public int Test()
+		{
+			return 6;
+		}
+	}
+
+	public class TestWithAfterAttribute
+	{
+		[After]
+		public static void Do()
+		{
+
+		}
+
+		[Attributes.Test]
+		public int Test()
+		{
+			return 6;
+		}
+	}
+	public class BadTestWithAfterAttribute
+	{
+		[After]
+		public static void Do()
+		{
+			throw new Exception();
+		}
+
+		[Attributes.Test]
+		public int Test()
+		{
+			return 6;
+		}
+	}
+
+	public class TestWithBeforeClassAttr
+	{
+		[BeforeClass]
+		public static void Do()
+		{
+
+		}
+
+		[Attributes.Test]
+		public int Test()
+		{
+			return 5;
+		}
+	}
+
+	public class BadTestWithBeforeClassAttr
```

Comment:
```suggestion
	public class BadTestWithBeforeClassAttribute
```
лучше не сокращать названия методов